### DISCUSSION TITLE
Timeline/ETA?

### DISCUSSION THREAD

**Discussion by Sternbach-Software** [cashapp/redwood#831] [Q&A]
I literally can't wait for this project to reach 1.0 (beta/stable)! I first heard about it in an amazing interview with Kotlin feat. @JakeWharton (star-struck btw 🤩 - huge long-time fan and admirer) I took a look at some of the issues, and it is very intriguing and exciting stuff! Is there a clear ETA for beta release, or at least a rough timeline of what needs to get done and how long it will take?

**Comment by colinrtwhite** [comment]
Redwood's still under active development so there aren't any ETAs to reach beta or stable. We do publish releases to Maven Central so if you'd like to try out Redwood you could depend on those, though the API is likely to change between releases. In terms of roadmap, I'd keep an eye on the [issues tracker](https://github.com/cashapp/redwood/issues) for some of our current priorities.

**Comment by AshleaPinntT** [comment]
Regarding your question, there's no clear ETA for Redwood's beta release yet, but don't worry, it's still under active development. In the meantime, you could try out the releases on Maven Central, although the API may change between releases.
